src/jit/hwintrinsicxarch.cpp

```diff
@@ -715,13 +715,13 @@ GenTree* Compiler::addRangeCheckIfNeeded(NamedIntrinsic intrinsic, GenTree* last
 // Arguments:
 //    isa - Instruction set
 // Return Value:
-//    true if
+//    true if EnableHWIntrinsic=true and
 //    - isa is a scalar ISA
 //    - isa is a SIMD ISA and featureSIMD=true
 //    - isa is fully implemented or EnableIncompleteISAClass=true
 bool Compiler::compSupportsHWIntrinsic(InstructionSet isa)
 {
-    return (featureSIMD || HWIntrinsicInfo::isScalarIsa(isa)) && (
+    return JitConfig.EnableHWIntrinsic() && (featureSIMD || HWIntrinsicInfo::isScalarIsa(isa)) && (
```

Comment:
The alternative to this is to make this part of the `getMaxIntrinsicSIMDVectorLength()` check which would result in `EnableHWIntrinsic` impacting whether the hardware acceleration or software fallback paths for `System.Numerics.Vectors` is chosen.